market_maker_keeper/dydx_market_maker_keeper.py

```diff
@@ -130,11 +138,11 @@ def place_order_function(new_order_to_be_placed):
         for new_order in new_orders:
             amount = new_order.pay_amount if new_order.is_sell else new_order.buy_amount
             side = 'Sell' if new_order.is_sell else 'Buy'
-            minimumOrderSize = float(self.market_info[self.pair().upper()]['minimumOrderSize'])
-            if self._should_place_order(new_order):
+            minimum_order_size = Wad.from_number(float(self.market_info[self.pair().upper()]['smallOrderThreshold']) / 10 ** 18)
```

Comment:
Why do we divide by 10^18 here?  `Wad.from_number` handles scaling.  Is the `market_info` request producing a scaled value?

Reply:
Yeah exactly, I need to divide by 10^18 in order to ensure `minimum_order_size` and `amount` are scaled equally. If the division is removed `40.000000000000000000` becomes `40000000000000000000.000000000000000000`

Reply:
It seems their API and Wad use the same scaling factor.  So, you divide by 10^18, and then `Wad.from_number()` multiplies by 10^18.  Since their API is providing a scaled value, I assume they are providing an integer, not a float.  Would the following be more readable?
```
minimum_order_size = Wad(int(self.market_info[self.pair().upper()]['smallOrderThreshold']))
```

Reply:
Ah, yeah that is way cleaner to avoid dividing then multiplying. Thank you for the suggestion, I will update accordingly.